### PR TITLE
Fix singular form in authentication doc

### PR DESCRIPTION
**Pull request by martinmaly**
The context deals with a single HTTP request and multiple
server plugins.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1682)

### CODE REVIEW

File: docs/admin/authentication.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ or be treated as an anonymous user.
 ## Authentication strategies
 
 Kubernetes uses client certificates, bearer tokens, or HTTP basic auth to
-authenticate API requests through authentication plugins. As HTTP request are
```
Comment (ericchiang) [inline docs/admin/authentication.md]: This should probably be

> As HTTP request_s_ are made to the API server, plugins attempt to associate the following attributes with the request.

Comment (martinmaly) [inline docs/admin/authentication.md]: Done. I hope GitHub will pick up the updated change. I also rebased on the current master.

**Comment by ericchiang** [comment]
thanks, lgtm.

cc @kubernetes/docs can someone merge?

**Comment by devin-donnelly** [comment]
This file has a merge conflict; I think somebody else was trying to fix the same typo.

**Comment by jaredbhatti** [comment]
I made the changes myself.  Closing this PR.  Thanks for the changes!

**Pull request closed without merging**